apollo-runtime/src/main/java/com/apollostack/android/GraphqlService.java

```diff
@@ -0,0 +1,14 @@
+package com.apollostack.android;
+
+import android.database.Observable;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/** A {@link retrofit2.Retrofit} service for executing GraphQL requests. */
+public interface GraphqlService {
```

Comment:
Oh, I just looked into the Retrofit documentation. My concern was that we needed the API Endpoint for the GraphQL requests, but that can be set as part of the Retrofit [`baseUrl`](http://square.github.io/retrofit/2.x/retrofit/retrofit2/Retrofit.Builder.html#baseUrl-okhttp3.HttpUrl-), so the API interface methods don't need to set the path themselves.
Alright, we're fine then. You're right that this should be only used internally.
I think we can have this be generated by the compiler as well, I'll make the changes.
One thing though is that we'll need some kind of plugin setting where the user will tell us if  they want to use RxJava or not, since that affects the generated code. Something like:
```groovy
apollo {
  useRxJava true
}
```
We could also add other plugin settings this way in the future.